editor/docks/scene_tree_dock.cpp

```diff
@@ -4028,9 +4023,6 @@ void SceneTreeDock::_update_tree_menu() {
 	PopupMenu *tree_menu = button_tree_menu->get_popup();
 	tree_menu->clear();
```

Comment:
I just realized that this menu doesn't need to be re-created every time. Only needs to update the checkboxes.
You could change it, but idk if it's within scope of this PR.

Reply:
No, no, you didn't just realise. [Older you did as well](https://github.com/godotengine/godot/pull/65932#discussion_r988452843). But the performance benefit [didn't outweigh the extra code at the time](https://github.com/godotengine/godot/pull/65932#discussion_r988453855).

Reply:
Well, now if there are no icons, there is no reason to clear it anymore.
I guess I will fix it together with auto-translation changes.